src/main/java/com/fontys_automotive/api/tus/models/project/Job.java

```diff
@@ -1,26 +1,36 @@
 package com.fontys_automotive.api.tus.models.project;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
 public class Job {
-        public String customer_id;
+        @JsonProperty("customer_id")
+        public String customerId;
         public String statusCode;
-        public boolean is_processed_internally;
+        @JsonProperty("is_processed_internally")
+        public boolean isProcessedInternally;
+        public String url;
         public String id;
         public String source;
 
-        public Job(String customer_id, String statusCode, boolean is_processed_internally, String id, String source, String creation_date, String import_date, String languageCode, Position position) {
-                this.customer_id = customer_id;
+        public Job(String customerId, String statusCode, boolean isProcessedInternally, String url, String id, String source, String creationDate, String import_date, String languageCode, Position position) {
```

Comment:
Rare positie voor een constructor tussen de properties door.

Deze constructor is overbodig aangezien hier gebruik gemaakt wordt van de `@Data` annotation

Reply:
Indien geen allArgsConstructor annotiatie aanwezig is, is een constructor gewoon nodig, positionering resolved